portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,33 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Andrew Hojel's Porfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="writeSnippets()">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <center>
+          <img id="profile" src="images/profile_picture.jpg"/>
+      </center>
+      <h1 class="roboto-font">👋 Hi, I am Andrew Hojel</h1>
+      <h3 id="text" class="roboto-font">I am a </h3>
+      <p class="roboto-font">
+          Hi! I am a rising junior at Stanford Univeristy studying Computer 
+          with a focus on Artificial Intelligence. My academic path furthers
+          my abilities as a builder, providing me the skills to seamlessly move
+          between ideation and implementation.
+      </p>
+      <p class="roboto-font">Click here to get a random fact about me:</p>
+      <button onclick="addRandomFact()">Hello</button>
+      <div id="fact-container"></div>
+      <div class="footer roboto-font">
```

Comment:
rather than adding the roboto-font class to everything, you can add a css rule for the entire page
something like:

body {
 font-family: 'Roboto Mono', monospace;
}
should do the trick